python/ray/rllib/agents/impala/vtrace.py

```diff
@@ -33,15 +33,17 @@
 from __future__ import print_function
 
 import collections
-
+from ray.rllib.models.action_dist import (Categorical, Deterministic,
```

Comment:
Can you break this PR into multiple smaller PRs?
1. Continuous Vtrace
2. MultiDiscrete cleanup
3. Other changes for APPO?

These have different levels of readiness / maturity and should be reviewed separately.